rosdep/base.yaml

```diff
@@ -287,7 +287,7 @@ boost:
     squeeze: [libboost1.42-all-dev]
     stretch: [libboost-all-dev]
     wheezy: [libboost-all-dev]
-  fedora: [boost-devel, boost-python2-devel]
+  fedora: [boost-devel, boost-python2-devel, boost-python3-devel]
```

Comment:
I see.  On Ubuntu bionic, `libboost-all-dev` pulls in `libboost-python-dev`, which pulls in `libboost-python1.65-dev`.  That package contains support for both python2 and python3, so adding in the additional key for Fedora makes sense.